CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -1,6 +1,18 @@
 # Everything beyond this point will be determined relatively
 # from this path.
 install_prefix="@CMAKE_INSTALL_PREFIX@"
+# Reset previous uvcdat env messing up
+source @CMAKE_INSTALL_PREFIX@/bin/reset_runtime.sh
+
+# Saves what we will mess with
+for v in PATH LD_LIBRARY_PATH DYLD_LIBRARY_PATH PYTHONPATH OPAL_PREFIX LIBOVERLAY_SCROLLBAR ; do
+   tmp="${v}"
+    if [ -n "${!tmp}" ] ; then
```

Comment:
I think you're right, let me just replace `source` with `.` as @aashish24 pointed out and ;et's do this. I tried the `eval` method pointed out on your link @sankhesh but it didn't seem to work when sourcing.